packages/coturn/project.bri

```diff
@@ -0,0 +1,55 @@
+import * as std from "std";
+import { cmakeBuild } from "cmake";
+import libevent from "libevent";
+import openssl from "openssl";
+
+export const project = {
+  name: "coturn",
+  version: "4.7.0",
+  repository: "https://github.com/coturn/coturn.git",
+};
+
+const source = Brioche.gitCheckout({
+  repository: project.repository,
+  ref: project.version,
+});
+
+export default function coturn(): std.Recipe<std.Directory> {
+  return (
+    cmakeBuild({
+      source,
+      dependencies: [std.toolchain, libevent, openssl],
+    })
+      // Fix broken absolute-path symlink
+      .insert("bin/turnadmin", std.symlink({ target: "turnserver" }))
```

Comment:
Not sure why, but the default output includes `turnadmin` as an absolute-path symlink to `turnserver`. This fixes it by using a relative-path symlink instead